backends/opentsdb/README.md

```diff
@@ -0,0 +1,72 @@
+# OpenTSDB with HTTP
+
+Since the version 1.16 the Netdata has the feature to communicate with OpenTSDB using HTTP API, to enable this channel
+it is necessary to set the following options in your netdata.conf
+
```

Comment:
Since the version 1.16 the Netdata -> Since version 1.16 Netdata

HTTP API, to -> HTTP API. To